includes/collection/class-replies.php

```diff
@@ -171,10 +173,16 @@ public static function get_context_collection( $post_id ) {
 		$post_uri = ( new PostTransformer( $post ) )->to_id();
 		\array_unshift( $ids, $post_uri );
 
+		if ( is_user_disabled( $post->post_author ) ) {
```

Comment:
I think we need to also check if the Blog User is enabled here. We could be in a situation where the Blog User is disabled, and this particular author is also disabled.

I guess we probably need to bail above in that case?

Reply:
yeah, feels like we need that check in more places. As in, we probably shouldn't offer the context in the first place in that case?

Reply:
ah, it only gets offered if this returns a response

Reply:
Fixed in [68d01ad](https://github.com/Automattic/wordpress-activitypub/pull/1427/commits/68d01adc88155be64f3b6e6e35544dd748820e65)